src/templates/project.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { graphql } from "gatsby";
+
+import styled from "styled-components";
+import Layout from "../components/Layout";
+import Button from "../components/Button";
+
+const Title = styled.h1``;
+const Client = styled.h2``;
+const Lead = styled.h3``;
+const Developers = styled.h3`
+  font-weight: normal;
+`;
+
+const Template = ({ data }) => {
+  const { markdownRemark } = data; // data.markdownRemark holds your post data
+  const { frontmatter, html } = markdownRemark;
```

Comment:
Can you destructure out the frontmatter props here? Like above, will make the rest of the code cleaner to read.